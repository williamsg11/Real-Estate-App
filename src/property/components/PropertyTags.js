import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

const PropertyTags = ({items}) => {
  return (
    <View style={styles.container}>
      {items.map((item, i) => (
        <Text key={i} style={styles.tag}> #{item} </Text>
      ))}
    </View>
  );
};

PropertyTags.propTypes = {
  items: React.PropTypes.array.isRequired,
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'row',
    paddingVertical: 2,
  },
  tag: {
    color: 'gray',
    fontSize: 11,
  },
});

export default PropertyTags;
