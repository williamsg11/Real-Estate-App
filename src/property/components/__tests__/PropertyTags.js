import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import PropertyTags from './../PropertyTags';

test('renders heart', () => {
  const tree = renderer
    .create(<PropertyTags items={['A', 'B', 'C']} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
