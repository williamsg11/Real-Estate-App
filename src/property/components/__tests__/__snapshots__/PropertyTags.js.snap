// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders heart 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "paddingVertical": 2,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }
  >
     #
    A
     
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }
  >
     #
    B
     
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }
  >
     #
    C
     
  </Text>
</View>
`;
